viewing_party/party.py

```diff
@@ -1,23 +1,291 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """ creating movie dictionary and return movies of dictionary """
+
+    # ditionary to store movies
+    movies = {}
+
+    # if any one of paramenters is None, function return None
+    if title == None or genre == None or rating == None:
+        return None
+
+    # add movie into dictionary, if all paramenter is valid value or truthy
+    elif title and genre and rating:
+        
+        movies["title"] = title
+        movies["genre"] = genre
+        movies["rating"] = rating
+        return movies
+    
+    # Otherwise, return None.
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    """ Adding movies into user data watched list and return user data"""
+
+    # list to store watched movies
+    watched_movies = []
+    # add watched movies to a list
+    watched_movies.append(movie)
+    # add watched movies list into user data watched list
+    user_data["watched"] = watched_movies
+    return user_data

```

Comment:
The logic of the function is great here! I think you don't need quite so many comments here and throughout your code though. It's already clear based off your great variable names what is going on in the code, the comments are a bit redundant.